ccdproc/combiner.py

```diff
@@ -150,15 +149,12 @@ def scaling(self):
 
         Parameters
         ----------
-
-        scale : function or array-like or None, optional
+        scale : function or `numpy.ndarray`-like or None, optional
```

Comment:
I think this is confusing since there is also a [`array.array`](https://docs.python.org/3.5/library/array.html#array.array). It may be obvious to everyone who had worked with numpy (and numpys documentation) but not necessarily anyone else. That was also part of the reason why I changed `ndarray` to `numpy.ndarray` and `MaskedArray` to `numpy.ma.MaskedArray`. It just differentiates more clearly between astropy-environment and externals.

This was a total shot in the dark but at least it was consistent. :smile: and having a consistent docstring allows simple search&replace. So if you want me to change it it's not much of a problem.